services/spar/src/Spar/Scim.hs

```diff
@@ -167,34 +169,26 @@ validateScimUser ScimTokenInfo{stiIdP} user = do
 --
 -- __Mapped fields:__
 --
---   * @userName@ is mapped to our 'userHandle'. If there is no handle, we use 'userId',
---     because having some unique @userName@ is a SCIM requirement.
---
---   * @name@ is left empty and is never stored, even when it's sent to us via SCIM.
+--   * @userName@ is mapped to our 'userHandle'.
```

Comment:
How is it mapped to userHandle, userHandle being globally unique? Perhaps explain or link to an explanation?

Reply:
This function doesn't do such checks. It only checks the schema.

Checks are done elsewhere when the user is created / updated.

I have updated the comment.

Reply:
If a handle is taken, the create request from the scim client will be answered with an error status?  What does the error look like?  Is that something that could use more haddocks?  (Not sure about any of this.)

Reply:
> Is that something that could use more haddocks?

Definitely.